src/systems/battery_plugin/LinearBatteryPlugin.cc

```diff
@@ -178,6 +178,9 @@ class gz::sim::systems::LinearBatteryPluginPrivate
 
   /// \brief Initial power load set trough config
   public: double initialPowerLoad = 0.0;
+
+  /// \brief Adjusts the sign of the current to align with ROS conventions.
+  public: bool adjustCurrentSignForROS{false};
```

Comment:
Do you mind to use a different value for the parameter ? We try not to include any ROS related stuff here.

Maybe something like `invertSignValues` ? But I open to suggestions

Reply:
go ahead with `invertCurrentSign` :ok_hand: